sources/HUBViewControllerExperimentalImplementation.m

```diff
@@ -716,11 +716,14 @@ - (void)collectionView:(UICollectionView *)collectionView
     forItemAtIndexPath:(NSIndexPath *)indexPath
 {
     HUBComponentWrapper * const componentWrapper = [self componentWrapperFromCell:(HUBComponentCollectionViewCell *)cell];
+    UIView * const componentView = componentWrapper.view;
 
-    [self.delegate viewController:self
-               componentWithModel:componentWrapper.model
-                     layoutTraits:componentWrapper.layoutTraits
-             didDisappearFromView:cell];
+    if (componentView) {
```

Comment:
If we're not expecting the view to be `nil`, maybe we should add an assert here?